packages/playground/src/weblets/tf_staticwebsite.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+  <weblet-layout
+    ref="layout"
+    :cpu="solution?.cpu"
+    :memory="solution?.memory"
+    :disk="solution?.disk"
+    :ipv4="ipv4"
+    :dedicated="dedicated"
+    :SelectedNode="selectionDetails?.node"
+    :valid-filters="selectionDetails?.validFilters"
+    title-image="images/icons/static_website.png"
+  >
+    <template #title>Deploy a Static Website Instance </template>
+
+    <form-validator v-model="valid">
+      <input-validator
+        :value="name"
+        :rules="[
+          validators.required('Name is required.'),
+          validators.isLowercase('Name should consist of lowercase letters only.'),
+          validators.isAlphanumeric('Name should consist of letters and numbers only.'),
+          name => validators.isAlpha('Name must start with alphabet char.')(name[0]),
+          validators.minLength('Name must be at least 2 characters.', 2),
+          validators.maxLength('Name cannot exceed 15 characters.', 15),
+        ]"
+        #="{ props }"
+      >
+        <input-tooltip tooltip="Instance name.">
+          <v-text-field label="Name" v-model="name" v-bind="props" />
+        </input-tooltip>
+      </input-validator>
+
+      <input-validator
+        :value="githubUrl"
+        :rules="[
+          validators.required('Github Repository is required.'),
+          validators.isURL('Github Repository must be a valid URL.', {
+            protocols: ['https'],
+          }),
+        ]"
+        #="{ props }"
+      >
+        <input-tooltip tooltip="Github Repository to serve.">
+          <v-text-field label="Github Repository" v-model="githubUrl" v-bind="props" />
+        </input-tooltip>
+      </input-validator>
+
+      <input-tooltip tooltip="Github Branch to serve (optional).">
+        <v-text-field label="Github Branch" v-model="githubBranch" />
+      </input-tooltip>
+      <input-validator :value="root" :rules="[validators.required('HTML Directory is required.')]" #="{ props }">
+        <input-tooltip
+          tooltip="HTML Directory to serve. Note that this field is case-sensitive so make sure you provide the correct directory"
```

Comment:
Please adjust so that it is not so long 
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/56790126/2f5515aa-f3d3-452c-872a-226d30c35761)

Reply:
Not relate to this pr, please open a new  issue for this